nirc_ehr/resources/queries/dbo/q_blood.sql

```diff
@@ -1,9 +1,15 @@
 
-SELECT anm.ANIMAL_ID_NUMBER AS Id,
-       CAST(anmEvt.EVENT_DATETIME AS TIMESTAMP) AS bloodDate,
+SELECT anmEvt.ANIMAL_EVENT_ID as objectid,
+       anm.ANIMAL_ID_NUMBER AS Id,
+       CAST(COALESCE (adt.CHANGE_DATETIME,anmEvt.EVENT_DATETIME) AS TIMESTAMP) AS bloodDate,
+       CAST(COALESCE (adt.CHANGE_DATETIME,anmEvt.EVENT_DATETIME) AS TIMESTAMP) AS modified,
```

Comment:
Does an audit event get written for inserts as well? I worry we'll get into a bad state if there are rows with a future EVENT_DATETIME that don't have an audit record. If this happened, we'd miss any new and updated rows until after that date.

Reply:
Looks like there is no inserts for Blood but there are for protocol data which is largely the data inside AUDIT_TRAIL table.

Reply:
Hmm... we need a different approach here. I think we need to use NOW() or something like that instead of anmEvt.EVENT_DATETIME. Let me know if you want to talk about the cases where this would fail to copy rows